packages/dd-trace/src/appsec/user_tracking.js

```diff
@@ -0,0 +1,168 @@
+'use strict'
+
+const crypto = require('crypto')
+const log = require('../log')
+const telemetry = require('./telemetry')
+const addresses = require('./addresses')
+const { keepTrace } = require('../priority_sampler')
+const { SAMPLING_MECHANISM_APPSEC } = require('../constants')
+const standalone = require('./standalone')
+const waf = require('./waf')
+
+// the RFC doesn't include '_id', but it's common in MongoDB
+const USER_ID_FIELDS = ['id', '_id', 'email', 'username', 'login', 'user']
+
+let collectionMode
```

Comment:
could it be assigned so you wouldn't need the `!collectionMode` in line 74?
```suggestion
let collectionMode = 'disabled'
```

Reply:
I don't think so, the reason why it's undefined at first is on purpose. When appsec/index.js calls this function, it must NOT overwrite remote config, and the way i did this was to only set the value if it's undefined. Because if it's not undefined, it was either set by RC or previously set by the same local config, in both cases we shouldn't override it.
TLDR it's to differentiate between "clean startup" and "remote config already set this value"